docs/mockups/ctevaluation-elcap.html

```diff
@@ -5368,158 +5392,568 @@ <h4>
      <input context="server" name="ceanod" type="hidden" value="n"/>
      <script>
       function initNoduleGrid(fv) {
-        const noduleArrayLength = parseInt("10");
-        const $noduleGrid = $.noduleGrid({
-            availableNodules: noduleArrayLength,
-            getNoduleCount: function () {
-                return parseInt($("[name='cennod']").val());
-            },
-            setNoduleCount: function (noduleCount) {
-                $("[name='cennod']").val(noduleCount);
-                $("[name='ceanod']").val(noduleCount === 0 ? "n" : "y");
-            }
-        });
+            const noduleArrayLength = parseInt("10");
+            const $noduleGrid = $.noduleGrid({
+                availableNodules: noduleArrayLength,
+                getNoduleCount: function () {
+                    return parseInt($("[name='cennod']").val());
+                },
+                setNoduleCount: function (noduleCount) {
+                    $("[name='cennod']").val(noduleCount);
+                    $("[name='ceanod']").val(noduleCount === 0 ? "n" : "y");
+                }
+            });
 
-        $(".add-nodule").on('click', $noduleGrid.addNodule);
-        $(".remove-nodule").on('click', $noduleGrid.removeNodule);
-        exclusiveCheckbox("[name='cectin']", "tr" /* the common parent of all checkboxes of this name*/);
+            $(".add-nodule").on('click', $noduleGrid.addNodule);
+            $(".remove-nodule").on('click', $noduleGrid.removeNodule);
+            exclusiveCheckbox("[name='cectin']", "tr" /* the common parent of all checkboxes of this name*/);
```

Comment:
It looks like you were adjusting the indentation here, but now the indentation level is 6 spaces for the first level, and only 4 spaces for the rest. I think you missed that the previous version was indented 2 spaces at the first level, and 4 spaces for the rest.

Maybe I'm way off here, but I think you want to make it consistent, so you probably just need to get rid of the first 2 spaces for each one of these lines.

Reply:
The indentation you see in these HTML files is irrelevant. These HTML files go through the Jinja templating engine and then the BeautifulSoup utility to make them "pretty" HTML 5 documents. The final format these are in is based on those utilities and has very little to do with the original source files. The particular line you noted is actually in _nodule_grid.jinja2 file. That is formatted using 4 spaces throughout and is very readable (IMHO). Though the tabbing isn't awfully important in the final HTML, I'll note that what is on each line **IS** important for the Mumps forms processor to work properly.